CMakeLists.txt

```diff
@@ -113,6 +117,8 @@ else()
     list(APPEND wxWidgetsComponentsRequired adv)
   endif()
 endif()
+
+message(STATUS "Finding: wxWidgets ver: ${wxWidgetsVersion}")
```

Comment:
This is superfluous and misleading, `find_package` does already output this, but only if it actually does search the package.